api/proto/teleport/devicetrust/v1/devicetrust_service.proto

```diff
@@ -266,6 +269,31 @@ message ListDevicesResponse {
   string next_page_token = 2;
 }
 
+// Request for ListDevicesByUser.
+//
+// Follows the pagination semantics of
+// https://cloud.google.com/apis/design/standard_methods#list.
+message ListDevicesByUserRequest {
+  reserved 3;
```

Comment:
Could you add comments explaining why the view is reserved here?  note it with the field only being available starting from Teleport 18 but it is not supported in Teleport 17 or earlier versions that's why the reserved exists here.

Can't we support the view at all?

Reply:
Its actually not used at all. I added it during the POC stage (the feature isn't released yet) but i couldn't remove the field completely. Should i say that in the comment?

Reply:
this is the correct protocol. I was just confused because the PR you link in the backport only mentions one the PRs